lighthouse/data-fixes/data_writers.py

```diff
@@ -0,0 +1,77 @@
+# take the CSV data and insert corrected IDs into the various DBs
+import pymongo
+import mysql.connector
+import pandas as pd
+import argparse
+
+from constants import (LOCALHOST, MONGO_DB, MONGO_DB_CLIENT, MONGO_TABLE, MYSQL_USER, MYSQL_PWD, MLWH_DB, fixed_samples_file)
+
+def write_data_to_db(data: pd.DataFrame, database: str):
+    if database.lower() == "mongo":
+        write_to_mongo(data)
+    elif database.lower() == "mysql":
+        write_to_mysql(data)
+    else:
+        print("Not a valid DB type")
+        return None
+
+def write_to_mongo(data):
+    print("Attempting to connect to Mongo DB...")
+    try:
+        client = pymongo.MongoClient(MONGO_DB_CLIENT)
+        db = client[MONGO_DB]
+        table = db[MONGO_TABLE]
+        print("Loading in the data...")
+        for index, row in data.iterrows():
+            root_sample_id = row["root_sample_id"]
+            original_root_sample_id = row["original_root_sample_id"]
+
+            update_query = { "Root Sample ID": original_root_sample_id }
+            new_value = { "$set": { "Root Sample ID": root_sample_id } }
+
+            table.update_one(update_query, new_value)
```

Comment:
I think this would be updateMany because we want updating all entries with the same root_sample_id.